schemas/cloudToDevice/wifi/wifi-position.json

```diff
@@ -0,0 +1,56 @@
+{
+    "title": "Wi-Fi Position Response",
+    "description": "Responds with coordinates",
+    "type": "object",
+    "properties": {
+        "appId": {
+            "type": "string",
+            "const": "WIFI"
+        },
+        "messageType": {
+            "type": "string",
+            "const": "DATA"
+        },
+        "data": {
+            "type": "object",
+            "properties": {
+                "lat": {
+                    "$ref": "#/definitions/Lat"
+                 },
+                 "lon": {
+                   "$ref": "#/definitions/Lon"
+                 },
+                 "uncertainty": {
+                   "$ref": "#/definitions/Uncertainty"
+                 }
+            },
+            "additionalProperties": false
+        },
+        "err": {
+            "type": "integer"
+        },
```

Comment:
This `properties` prop should only include appId, messageType, data, and err. The object should be closed and then have additionalProperties, required, definitions outside of it. That will fix the tests.

Reply:
It's a really subtle change that took me a while to figure out because it is nearly identical to cell position and it's valid JSON. Really hard to detect.